resources/views/inputs/input-with-icon.blade.php

```diff
@@ -21,6 +21,7 @@ class="{{ $inputClass ?? '' }} input-text-with-icon @error($name) input-text-wit
                     @if($readonly ?? false) readonly @endif
                     @isset($inputMode) inputmode="{{ $inputMode }}" @endisset
                     @isset($pattern) pattern="{{ $pattern }}" @endisset
+                    @if($onAnyKeydown ?? false) @keydown="{{ $onAnyKeydown }}" @endif
```

Comment:
use `isset` as that's less `??????` stuff + preferably rewrite the component to make use of attributes: https://laravel.com/docs/8.x/blade#managing-attributes